### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by alizadamuhammmed**

### CODE REVIEW

File: Muhammed.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+public class Muhammed {
+    public static void main(String[] args) {
+        //Car
+        String model = "Toyota Supra";
+        byte numberOfDoors = 2;
+        String color = "Black";
+        long price;
+        short dateOfProduction = 2017;
+        boolean crash = false;
+        int distanceKM = 232545;
```
Comment (sabuhihuseynov) [inline Muhammed.java:10]: Based on naming convention using two upper case letter consecutive is not desirable. instead you can use distanceKm

File: Muhammed.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        int lengthMM;
+        short productionYear = 2012;
+        String color2 = "White";
+        //TV
+        String brand = "LG";
+        String modelName;
+        long serialID;
+        short screenSizeWithInches;
+        boolean smart = true;
+        short ProductionYear2 = 2021;
```
Comment (sabuhihuseynov) [inline Muhammed.java:35]: Variable should start with lowerCase

File: Muhammed.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        // Phone
+        String brand2 = "Apple";
+        String model2 = "Iphone 13";
+        byte Cameras = 2;
```
Comment (sabuhihuseynov) [inline Muhammed.java:39]: also lowerCase

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
Pls look

File: Muhammed.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        short pageCount; // Longest book in the world is One Piece (21450 pages)
+        String author = "George Orwell";
+        int priceDollar = 12;
+        String type;
+        short publicationYear = 1939;
+        String title;
+        //Student
+        String name = "Muhammed";
+        byte classYear = 11;
+        long idOfStudent;
```
Comment (sabuhihuseynov) [inline Muhammed.java:21]: you can use only `id `

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

**Comment by soltankara** [comment]
@alizadamuhammmed you are supposed to clone this repo and create a branch on local. Then push your branch to this repo. Only after that a PR should be created.

**Pull request closed without merging**